docs/domains/Domain.json

```diff
@@ -296,6 +296,10 @@
           "description": "Specified whether the log home folder is enabled. Not required. Defaults to true if domainHomeInImage is false. Defaults to false if domainHomeInImage is true. ",
           "type": "boolean"
         },
+        "httpAccessLogInLogHome": {
+          "description": "If true (the default), the server HTTP access log file will be written to the same directory specified in logHome. Otherwise, server HTTP access log file will be written to the directory configured in the WebLogic domain home configuration.",
```

Comment:
the server HTTP --> a 
server HTTP log file will --> server HTTP log files will

Reply:
(the default), --> (the default), then
(I'm channeling Rosemary on that one -- any 'if' needs a corresponding 'then' in our doc.  It helps folks that are new to English.)